src/ModelParameters.jl

```diff
@@ -14,6 +14,12 @@ import AbstractNumbers,
 
 using Setfield
 
+using PackageExtensionCompat
+
+function __init__()
```

Comment:
Hmm I don't love `__init__()` blocks... can we just minimum bound to 1.10? It will be the LTS version very soon.